zulipterminal/ui_tools/boxes.py

```diff
@@ -113,7 +113,7 @@ def autocomplete_mentions(self, text: str, prefix_string: str
                               ) -> List[str]:
         # Handles user mentions (@ mentions and silent mentions)
         # and group mentions.
-        group_typeahead = ['@*{}*'.format(group_name)
+        group_typeahead = [prefix_string + '*{}*'.format(group_name)
                            for group_name in self.model.user_group_names
                            if match_group(group_name, text[1:])]
```

Comment:
This worked and didn't show any suggestion for `@_` or silent group mentions because `match_group` received `text[1:]` instead of `text[len(prefix_string):]`.

However, on second thought, the existing code, with hard-coded `@`, seems clearer and doesn't have any potential anomaly. I'll shelve this commit.